bokehjs/src/coffee/main.coffee

```diff
@@ -16,7 +16,10 @@ if not window.Float64Array
 
 # binding the libs that bokeh uses so others can reference them
 Bokeh._                 = require("underscore")
-Bokeh.$ = $ = jQuery    = require("jquery")
+if $ || jQuery
+    Bokeh.$ = $ = jQuery = require("jquery").noConflict(true)
+else
+    Bokeh.$ = $ = jQuery = require("jquery")
```

Comment:
> So are we talking about removing the `= $ = jQuery`?

Yes, because those are unnecessary local aliases.

> In that case I wouldn't think we need `noConflict()`.

 No, because although `$` and `jQuery` are behind a `var` (it's coffeescript), jquery will still do `window.$ = ...` and `window.jQuery = ...`. `noConflict()` just restores the original globals from jquery's cached values (or undefined if no jquery was loaded before) (see https://github.com/jquery/jquery/blob/master/src/exports/global.js).